library/java/net/openid/appauth/AppAuthConfiguration.java

```diff
@@ -94,12 +114,33 @@ public Builder setConnectionBuilder(@NonNull ConnectionBuilder connectionBuilder
             return this;
         }
 
+        /**
+         * Disables https validation for the issuer identifier.
+         */
+        public Builder setSkipIssuerHttpsCheck(Boolean skipIssuerHttpsCheck) {
+            mSkipIssuerHttpsCheck = skipIssuerHttpsCheck;
+            return this;
+        }
+
+        /**
+         * Disables nonce verification for value sent in the Authentication Request.
```

Comment:
Could also link to a reasoning on why this *should* be enabled, even if it's not the most critical thing.